paper/paper.md

```diff
@@ -0,0 +1,198 @@
+    orchid: 0009-0000-9432-9795
+    affiliation: 2
+  - name: Zezhi Wang 
+    orchid: 0000-0001-6988-5853
+    affiliation: 9
+  - name: Damir Pulatov
+    orchid: 0000-0003-4901-7201
+    affiliation: 14
+  - name: Keenan Ganz
+    orchid: 0000-0002-8486-3959
+    affiliation: 10
+  - name: Henri Funk
+    orchid: 0009-0007-0949-8385
+    affiliation: "3, 11, 12"
+  - name: Liana Harutyunyan
+    affiliation: 17
+  - name: Pierre Camilleri
+    orchid: 0009-0005-1070-0670
+    affiliation: 16
+  - name: Philipp Kopper
+    orchid: 0000-0002-5037-7135
+    affiliation: 3
+  - name: Andreas Bender
+    orchid: 0000-0001-5628-8611
+    affiliation: "2, 3"
+  - name: Bernd Bischl
+    orcid: 0000-0001-6002-6980
+    affiliation: "2, 3"
+affiliations:
+ - name: TU Dortmund University, Germany
+   index: 1
+ - name: Department of Statistics, LMU Munich, Germany
+   index: 2
+ - name: Munich Center for Machine Learning (MCML), Germany
+   index: 3
+ - name: Department of Cancer Genetics, Institute for Cancer Research, Oslo University Hospital, Norway
+   index: 4
+ - name: Leibniz Institute for Prevention Research and Epidemiology (BIPS), Bremen, Germany
+   index: 5
+ - name: Faculty of Mathematics and Computer Science, University of Bremen, Germany
+   index: 6
+ - name: Certilytics, Inc., 9200 Shelbyville Rd, Louisville, KY, 40222, USA
+   index: 7
+ - name: Friedrich-Alexander-Universität Erlangen-Nürnberg (FAU), Erlangen, Germany
+   index: 8
+ - name: Department of Statistics and Finance/International Institute of Finance, School of Management, University of Science and Technology of China, Hefei, Anhui, China
+   index: 9
+ - name: School of Environmental and Forest Sciences, University of Washington, Seattle
+   index: 10
+ - name: Department of Geography, LMU Munich, Germany
+   index: 11
+ - name: Statistical Consulting Unit StaBLab, LMU Munich, Germany
+   index: 12
+ - name: Wake Forest University School of Medicine, Department of Biostatistics and Data Science, Division of Public Health SciencesWinston-Salem, North Carolina
+   index: 13
+ - name: University of North Carolina Wilmington
+   index: 14
+ - name: OSPO Now
+   index: 15
+ - name: multi, 8 passage Brûlon, 75012 PARIS, France
+   index: 16
+ - name: ServiceTitan, Inc., Glendale, California
+   index: 17
+date: XXX December 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+The [`mlr3extralearners`](https://mlr3extralearners.mlr-org.com/) [`R`](https://www.r-project.org/) [@R] package is a community-driven package that integrates external machine learning algorithms into the [`mlr3`](https://mlr3.mlr-org.com/) [@Lang2019] ecosystem.
+The `mlr3` ecosystem is a versatile toolbox for machine learning in `R` and is targeted towards both practitioners and researchers [@Bischl2024].
+At its core, the `mlr3extralearners` package provides a standardized interface for machine learning and connects many R packages implementing machine learning algorithms into a unified framework.
```

Comment:
I wouldn't say that this is what "mlr3extralearners" does. The package offering a standardized interface is mlr3. mlr3extralearners is a repository of connectors that integrate existing ML packages into mlr3.